examples/use-cases/src/encrypt.ts

```diff
@@ -0,0 +1,43 @@
+import { VaultClient } from "@piiano/vault-client";
+
+async function main() {
+  const client = new VaultClient({
+    vaultURL: "http://localhost:8123",
+    apiKey: "pvaultauth",
+  });
```

Comment:
`http://localhost:8123` and `pvaultauth` are the defaults so you can change to:
```suggestion
  const client = new VaultClient();
```

Reply:
Wasn't sure that's the default. Great, will try it.